parlai/crowdsourcing/tasks/acute_eval/acute_eval_blueprint.py

```diff
@@ -174,12 +136,12 @@ def get_initialization_data(self) -> Iterable["InitializationData"]:
         # TODO(#99) once we can release HITs over time, configure this to
         # release as many as needed thusfar and top off when
         # onboardings fail
-        print(self.opts)
+        print(self.args)
```

Comment:
I think this was leftover debug, idk if we still want it.

Reply:
Good point, removed